JustFakeIt/HttpRequestExpectation.cs

```diff
@@ -5,6 +5,8 @@ namespace JustFakeIt
 {
     public class HttpRequestExpectation
     {
+        public IBodyMatchingOption BodyMatching;
```

Comment:
Why is this a member, rather than a property? Is it meant to be mutable to the world?